app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> str:
+    not_wearing_mask_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately upon encountering a `VaccineError` means that the function will not check the remaining friends. If the intention is to check all friends before returning a message, consider accumulating errors and returning a summary after the loop.